terraform/modules/configs/deploy-all/main.tf

```diff
@@ -102,6 +102,10 @@ data "aws_ssm_parameter" "spree_db_password" {
   with_decryption = true
 }
 
+data "aws_ssm_parameter" "elasticsearch_url" {
+  name = "/bat/sbx1-elasticsearch-url"
```

Comment:
Hardcoded environment - replace with `${lower(var.environment)}`

Reply:
Should be fixed now 👍